zproject/settings.py

```diff
@@ -1272,6 +1272,7 @@ def get_dirs(self):
 
 # iOS App IDs
 ZULIP_IOS_APP_ID = 'org.zulip.Zulip'
+APNS_TOPIC="xxx"
```

Comment:
See https://zulip.readthedocs.io/en/latest/subsystems/settings.html; the right way to do this is to set `APNS_TOPIC` and `DEFAULT_SETTINGS` and then just accessing `settings.APNS_TOPIC` from the code; then anyone can override in `/etc/zulip/settings.py` without any code changes to Zulip.

Reply:
I have made changes as per your suggestion. Please review and let me know if anything else is required from my side. Thanks for properly guiding me through this.